tests/components/asuswrt/test_sensor.py

```diff
@@ -61,23 +60,17 @@ async def test_sensors(hass: HomeAssistant):
             return_value=MOCK_CURRENT_TRANSFER_RATES
         )
 
-        now = datetime(2020, 1, 1, 1, tzinfo=dt_util.UTC)
-        with patch(("homeassistant.helpers.event.dt_util.utcnow"), return_value=now):
-            assert await async_setup_component(hass, DOMAIN, VALID_CONFIG_ROUTER_SSH)
-            await hass.async_block_till_done()
-            async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
-            await hass.async_block_till_done()
+        assert await async_setup_component(hass, DOMAIN, VALID_CONFIG_ROUTER_SSH)
+        await hass.async_block_till_done()
+        async_fire_time_changed(hass, utcnow() + timedelta(seconds=30))
+        await hass.async_block_till_done()
 
-            assert (
-                hass.states.get(f"{sensor.DOMAIN}.asuswrt_devices_connected").state
-                == "3"
-            )
-            assert (
-                hass.states.get(f"{sensor.DOMAIN}.asuswrt_download_speed").state
-                == "160.0"
-            )
-            assert hass.states.get(f"{sensor.DOMAIN}.asuswrt_download").state == "60.0"
-            assert (
-                hass.states.get(f"{sensor.DOMAIN}.asuswrt_upload_speed").state == "80.0"
-            )
-            assert hass.states.get(f"{sensor.DOMAIN}.asuswrt_upload").state == "50.0"
+        assert (
+            hass.states.get(f"{sensor.DOMAIN}.asuswrt_devices_connected").state == "3"
+        )
+        assert (
+            hass.states.get(f"{sensor.DOMAIN}.asuswrt_download_speed").state == "160.0"
+        )
+        assert hass.states.get(f"{sensor.DOMAIN}.asuswrt_download").state == "60.0"
+        assert hass.states.get(f"{sensor.DOMAIN}.asuswrt_upload_speed").state == "80.0"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)